document-generator-interfaces/src/main/java/uk/gov/companieshouse/document/generator/interfaces/model/DocumentInfo.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.companieshouse.document.generator.interfaces.model;
+
+import com.google.gson.Gson;
+
+import java.util.Map;
+
+public class DocumentInfo {
+    
+    String data;
+    String templateName;
+    String assetId;
+    String location;
+    String Description;
+    String DescriptionIdentifier;
```

Comment:
These shouldn't be capitalised. Can you make the variables private to encapsulate them too please?